README.md

```diff
@@ -17,7 +17,7 @@ We also assume that you're deploying an application behind an ALB to port 443 (a
 
 ```hcl-terraform
 module "terraform-ecs-task" {
-  source                = "git::https://github.com/7Factor/terraform-ecs-task.git"
+  source                = "github.com/7factor/terraform-ecs-http-task?ref=v1.0.0"
```

Comment:
I changed the format because I recall some issue with using tags with the previous format here.